### DISCUSSION TITLE
Bucket notification

### DISCUSSION THREAD

**Discussion by BlackWand1** [minio/minio#18972] [Q&A] [answered]
Hey!

I have been trying to configure bucket notification to be able to get better up-to-date values regarding the size of every bucket.
First off the guide for this could have been more detailed. For example what steps need to be taken in the postgres server.

Anywho. Once I got to the part where I can actually activate bucket notification, it looks like it can only be done on a PER bucket config?
Or am I missing something. I want it active for every bucket.

Would that require that I activate it for each bucket manually?

**Comment by klauspost** [comment, marked as answer]
Yes, "bucket notifications" are per bucket. You can use `mc` to easily set up multiple buckets - by for example writing a script: https://min.io/docs/minio/kubernetes/upstream/administration/monitoring/publish-events-to-postgresql.html#configure-bucket-notifications-using-the-postgresql-endpoint-as-a-target

If you need further guidance than what our documentation provides you are welcome to sign up for our [subscription](https://min.io/pricing), where we can give personal assistance.

**Comment by mctoohey** [comment]
On your point of wanting to activate bucket notifications for every bucket. You could use the s3:BucketCreated https://min.io/docs/minio/linux/administration/monitoring/bucket-notifications.html event to trigger a script to add your notification subscription as soon as a bucket is created.
